pages/background.md

```diff
@@ -1,25 +1,30 @@
+{% include nav.html %} {% include css.html %}
+
 # Project Background
 
 ---
 
-Put a general discussion of your topic here. Don't get into the historiography or scholarship here. Just introduce to your reader what your topic is. What is the 'problem', the 'question', and/or the 'argument'.
+The Triangular trade was by far one of the worlds most important trade route. The route funnelled the foundations of slave labor into much of the carribean and the southern United States. However, has the slave trade ever been infiltrated by acts of piracy, and to what extent did piracy hinder the progression of the Triangular trade? With our thesis in tow, we aim to understand the economic effect piracy had on Slave Voyages.
 
-YOU SHOULD BE LIBERAL AND INCLUDE ANY AND ALL LINKS WHEN RELEVANT
 
 ---
 
+![image1]({{ site.baseurl }}/assets/images/background picture.jpg)
+
 ## Historical Background
 
-This is where your historiography goes. You can discuss the historiography of specific times and places, but you can also discuss any theoretical literature you think is relevant.
+Although the history of piracy is often considered small than that of the Trans-Atlantic triangular trade, the social and cultural implications of piracy shaped many of today’s understanding of pirates and their impact on the triangular trade economically. However, historians that study piracy today often study the understanding of pirates in regard to social and cultural movements. In the mid-to-late- 1900s new cultural waves of understanding clouded academia and led to some of the worlds most influential writers of pirate history. Historians of the social turn inspired by new movements for civil rights and women’s rights, formulate new lenses to view events in the past.
+
```

Comment:
Again, citations should be here. I am going to stop mentioning citations from here on out, but fixing that up would make it much more academic and professional for a future portfolio.